testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/support/support_logging.t.js

```diff
@@ -0,0 +1,139 @@
+          args: 'nx-message-notification',
+          desc: 'Wait for message dialogs to close'
+        }
+      };
+
+  Ext.merge(t.controller('Message').windowOptions, {
+    slideInDuration: -1,
+    slideBackDuration: -1,
+    autoCloseDelay: -1,
+  });
+
+  t.describe('Support Logging', function(t) {
+    t.it('Logging Smoke Test', function(t) {
+      t.chain(
+        t.openPageAsAdmin('admin/support/logging'),
+        { waitFor: 'CQVisible', args: 'nx-coreui-logger-list' },
+        { waitFor: 'CQVisible', args: 'nx-coreui-logger-list button[action=new]' },
+        { waitFor: 'CQVisible', args: 'nx-coreui-logger-list button[action=delete]' },
+        { waitFor: 'CQVisible', args: 'nx-coreui-logger-list button[action=reset]' },
+        waitForMessageDialogsToClose(),
+        { waitForRowsVisible: '>>nx-coreui-logger-list' },
+        { waitFor: function() { return t.cq1('nx-coreui-logger-list'); } },
+        { waitFor: function() { return t.cq1('nx-coreui-logger-list').getStore(); } },
+        clickRow(t, 'nx-coreui-logger-list', 'name', 'ROOT'),
+        { click: 'combobox[dataIndex=level] => .x-form-text' },
+        { click: '.x-boundlist-item:contains(DEBUG)' },
+        { click: '>>button#cancel' },
+        // explore reset to default levels
+        { click: '>>nx-coreui-logger-list button[action=reset]' },
+        { waitForCQVisible: 'button#yes' },
+        function(next, components) { t.click(components[0], next); },
+        waitForMessageDialogsToClose()
+      );
+    });
+    t.it('Create Logger', function(t) {
+      t.chain(
+        // create logger {newLoggerName}
+        { waitForRowsVisible: '>>nx-coreui-logger-list' },
+        { click : '>>nx-coreui-logger-list button[action=new]' },
+        { type: newLoggerName, target: '>>nx-coreui-logger-add field#name' },
+        { waitForCQVisible: 'nx-coreui-logger-add button[action=add]' },
+        function(next, components) { t.click(components[0], next); },
+        waitForMessageDialogsToClose(),
+        // look for {newLoggerName} in the grid panel
+        { waitForRowsVisible: '>>nx-coreui-logger-list' },
+        function(next) {
+          var grid = t.cq1('nx-coreui-logger-list'),
+              store = grid.getStore(),
+              model = store.findRecord('name', newLoggerName);
+          t.ok(model, 'Logger "' + newLoggerName + '" exists');
+          t.is(model.get('level'), 'INFO', 'Logger level is INFO');
+          next();
+        }
+      );
+    });
+    t.it('Update Logger', function(t) {
+      t.chain(
+        // update logging level for {newLoggerName}
+        { waitForRowsVisible: '>>nx-coreui-logger-list' },
+        clickRow(t, 'nx-coreui-logger-list', 'name', newLoggerName),
+        { click: 'combobox[dataIndex=level] => .x-form-text' },
+        { click: '.x-boundlist-item:contains(ERROR)' },
+        { waitForCQVisible: 'button#update' },
+        function(next, components) { t.click(components[0], next); },
```

Comment:
The crux of the failure under PhantomJS appears to the fact that the simulated click doesn't think the button is visible even though the `waitForCQVisible` just found it. The following is from a run via maven/PhantomJS:
```
Received command: log:ok 3 - Waited 0 ms for  componentQuery: button#update to return a visible component
ok 3 - Waited 0 ms for  componentQuery: button#update to return a visible component
Received command: keepAlive
Received command: log:[WARN] # getNormalizedTopElementInfo: Target element of action [simulateMouseClick] is not visible or not reachable: #button-1235
[WARN] # getNormalizedTopElementInfo: Target element of action [simulateMouseClick] is not visible or not reachable: #button-1235
```